ui/src/app/components/sidenav-save-session/sidenav-save-session.component.ts

```diff
@@ -38,7 +38,7 @@ export class SidenavSaveSessionComponent implements OnInit {
       SessionName: formValue.SessionName.trim(),
       EditorName: formValue.EditorName.trim(),
       DatabaseName: formValue.DatabaseName.trim(),
-      Notes: formValue.Notes.split('\n'),
+      Notes: formValue.Notes?.split('\n'),
```

Comment:
In case of empty notes while saving the session its showing the following error which shouldn't happen because its an optional field: https://drive.google.com/file/d/14_Tr9vcuJ2rN-IKMffyiesSm9ivJRCdX/view?usp=sharing&resourcekey=0-lRXWHZmSwMkETUgEWnbvSA

Reply:
Metadata table schema updated with Notes and EditorName as nullable field. Please delete the old metadata database so as to create a new database with updated schema.